assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { dataFontEm } from '../../../../units';
+
+const DEFAULT_PRESET = {
+  id: 'default-text',
+  content: __('Fill in some text', 'web-stories'),
+  fontSize: dataFontEm(1),
+  fontWeight: 400,
+  font: {
+    family: 'Roboto',
+    service: 'fonts.google.com',
+  },
+  lineHeight: '24px',
```

Comment:
Note that the `lineHeight` used for text element is not in pixels, it's just a number. So this should probably be calculated based on the font size to get `24px`.

Reply:
Correct. I believe it's still a fractional value. But please double check. There was a plan to switch it to a percent value.

Reply:
Though, why do we need it here at all? Why not just go with `auto`?

Reply:
This is due to the UX requirement in the bug and in figma of a 24px line height. Since 1em is 16px, then it should be 1.5

Reply:
Ok, I see. This is conflated with the https://github.com/google/web-stories-wp/issues/1820 issue, which is fairly broader. I recommend dropping `lineHeight` from here (e.g. assuming the auto value will be addressed at some point) and instead focusing on other nuances.

Reply:
Removed.